target/linux/ath79/files/drivers/mfd/rb4xx-cpld.c

```diff
@@ -151,10 +151,17 @@ static int rb4xx_cpld_probe(struct spi_device *spi)
 				    NULL, 0, NULL);
 }
 
+#if LINUX_VERSION_CODE > KERNEL_VERSION(5,17,0)
+static void rb4xx_cpld_remove(struct spi_device *spi)
```

Comment:
Why not just drop the remove method completely as it does nothing anyway?

Reply:
Just checked the driver, yes can be done.